bpf/cpu/cpu.bpf.c

```diff
@@ -121,7 +123,6 @@ typedef struct shard_info {
 
 // Unwind table shards for an executable mapping.
 typedef struct stack_unwind_table_shards {
-  u64 len;
```

Comment:
We don't need this anymore. We traverse all the chunks until one matches. Calculating the total amount of chunks ahead of time complicates things and doesn't yield any tangible benefit.